src/components/v2/V2Project/NftRewardsSection/index.tsx

```diff
@@ -58,6 +69,20 @@ export function NftRewardsSection({
 
     const nextRewardTier = rewardTiers[index + 1]
 
+    const handleSelected = () => {
+      setSelectedIndex(index)
+      // update pay input with NFT's contribution floor
+      onPayAmountChange(
+        (payInCurrency === USD // convert to USD if USD selected in pay input
+          ? converter
+              .weiToUsd(parseWad(rewardTier.contributionFloor))
+              ?.add(1) //-> adds 1USD to ensure ETH amount is above the contributionFloor
```

Comment:
ETH can fluctuate way more than $1 very quickly. We should likely add a warning saying something like:
> Paying in USD to secure an NFT can be risky due to volatility. If you want to ensure you secure the NFT, we recommend paying in ETH instead.

And we have this setup to only appear when the user is currently paying enough to receive an NFT.

WDYT?

Reply:
maybe not saying "paying in eth" but smoething along those lines to convey using eth rather than the converted usd amount

Reply:
does it even make sense for tiers to be denominated in USD?

Should we just not allow USD selection if NFT reward selected?

Reply:
Thing is, they get auto selected so you can’t exactly disable it when an NFT is chosen